### DISCUSSION TITLE
Allowing projects with large files to be saved

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#6883] [Ideas] [answered]
To avoid this problem https://scratch.mit.edu/projects/869052380 could large files be split when they are loaded in Scratch (with a warning dialog ok/cancel that explains what will be done?)

**Comment by CubesterYT** [comment]
Maybe have a system that detects if the file is to large, notify the user, and give them the option to discard, use anyways, or split it up for them (for the audio).

**Comment by DNin01** [comment, marked as answer]
Related PR: #6507

**Reply by WorldLanguages** [reply]
Yeah, we should highlight any sounds that exceed the limit. It's also probably feasible to split it automatically using the built-in sound module of Scratch.

**Reply by sfederici** [reply]
It would be similar to when someone loads an animated GIF and finds in the costume tab a full list of new costumes.
